int64 reqtofanuc
int64 posnumber
int64 ionumber
float64 x
float64 y
float64 z
float64 j1
float64 j2
float64 j3
float64 j4
float64 j5
float64 j6
float64 sx
float64 sy
float64 sz
float64 sr
---
int64 respfromfanuc
